src/test/java/net/snowflake/client/log/JDK14LoggerWithClientLatestIT.java

```diff
@@ -89,4 +90,60 @@ public void testJDK14LoggerWithQuotesInMessage() {
     logger.debug("Returning column: 12: a: Group b) Hi {Hello 'World' War} cant wait");
     JDK14Logger.setLevel(Level.OFF);
   }
+
+  @Test
+  public void testJDK14LoggingWithMissingLogPathClientConfig() {
+    Path configFilePath = Paths.get("config.json");
+    String configJson = "{\"common\":{\"log_level\":\"debug\"}}";
+    try {
+      Files.write(configFilePath, configJson.getBytes());
+      Properties properties = new Properties();
+      properties.put("client_config_file", configFilePath.toString());
+      Connection connection = getConnection(properties);
```

Comment:
let's create connection, statement and result set in try with resources - also in other added tests